libsolidity/experimental/ast/CallGraph.cpp

```diff
@@ -0,0 +1,39 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+// SPDX-License-Identifier: GPL-3.0
+
+#include <libsolidity/experimental/ast/CallGraph.h>
+
+using namespace solidity::frontend::experimental;
+
+// TODO remove before merge (delete entire file)
+std::ostream& solidity::frontend::experimental::operator<<(std::ostream& _out, CallGraph const& _callGraph)
```

Comment:
Huh, I was actually looking into Rodrigo's [Lengauer/Tarjan PR](https://github.com/ethereum/solidity/pull/14460/files#diff-c8f1f453a4e47ba42728eda013ff9278fbb8568c937913e68413e6ae987e07fe), but this looks even more convenient.

Reply:
By the way, not sure if it makes sense to do this now , because technically, this is only the first portion of the feature - the actual goal is to take this graph, break it into strongly connected components, and then create a forest of them, which is what will ultimately be used in the type inference. @ekpyron should I even bother with this prior to implementing the forest of SCCs?